src/entity/User.ts

```diff
@@ -183,6 +185,7 @@ export const addNewUser = async (
   logger: FastifyLoggerInstance,
 ): Promise<AddNewUserResult> => {
   if (!checkRequiredFields(data)) {
+    console.log(data);
```

Comment:
if it's a proper log, you should use the `logger` object. Otherwise please remove it